topo_processor/file_system/transfer.py

```diff
@@ -1,11 +1,13 @@
+from typing import Union
+
 from linz_logger import get_log
 
-from topo_processor.util import time_in_ms
+from topo_processor.util.time import time_in_ms
 
 from .get_fs import get_fs
 
 
-def transfer_file(source_file: str, checksum: str, content_type: str, target_file: str) -> None:
+def transfer_file(source_file: str, checksum: str, content_type: Union[str, None], target_file: str) -> None:
```

Comment:
There seems to be two options for when variables could be none:
- `Union[str, None]`
- `Optional[str]`

Which one is best? (should we be consistent because I've used `Optional[str]`?)

Reply:
It does the same thing (https://docs.python.org/3/library/typing.html#typing.Optional). I vote for `Optional`.

Reply:
Also pose the question of `A | B` or `Union[A, B]` 

`A | B` looks better to me as its what typescript uses :)

Reply:
I think it is not on the current version we use though

Reply:
That's actually the reason I asked about Python versions earlier this week :)

Reply:
If we plan to move to `3.10` one day, should we leave this change until then?

Reply:
ahh ok! didn't realise it was that new...